addons/vela-workflow/parameter.cue

```diff
@@ -9,7 +9,7 @@ const: {
 
 parameter: {
 	image:                *"oamdev/vela-workflow" | string
-	version:              *"0.3.5" | string
+	version:              *"0.4.0-alpha.1" | string
```

Comment:
The workflow controller is alpha, set the addon version to alpha better. Do you want to the user default install this version?

Reply:
The official v4.0 is just released, I'll change to version to the official one.